s3path.py

```diff
@@ -16,11 +16,14 @@
     from botocore.exceptions import ClientError
     from botocore.docs.docstring import LazyLoadedDocstring
     import smart_open
+    from s3transfer.manager import TransferManager
```

Comment:
do you want to add dependency with s3transfer package?
Can you explain?

Reply:
There is already an implicit dependency on s3transfer via boto3:
https://github.com/boto/boto3/blob/develop/setup.cfg